src/libs/ajax.js

```diff
@@ -1232,11 +1235,17 @@ const Apps = signal => ({
       _.mergeAll([authOpts(), appIdentifier, { signal }]))
     return res.json()
   },
+  listWithoutProject: async (labels = {}) => {
+    const res = await fetchLeo(`api/google/v1/apps?${qs.stringify(labels)}`,
+      _.mergeAll([authOpts(), appIdentifier, { signal }]))
+    const resFinal = res.json()
+    return resFinal
```

Comment:
```suggestion
    return res.json()
```

Reply:
oops meant to remove this, did it for testing purposes